hhs_hosp/tests/test_run.py

```diff
@@ -54,6 +54,7 @@ def test_make_signal():
     data = pd.DataFrame({
         'state': ['NA'],
         'date': [20200102],
+        'population':[100.],
```

Comment:
looks like this column is never used -- be sure to test expected `make_signal` output for all signals.